### PR TITLE
js: Follow Serenity C++ naming conventions

### PR DESCRIPTION
**Pull request by linusg**
Am I understanding https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#names correctly? :^)

### CODE REVIEW

File: Userland/js.cpp [hunk 1]
```diff
@@ -62,10 +62,10 @@ class ReplObject : public JS::GlobalObject {
     static JS::Value save_to_file(JS::Interpreter&);
 };
 
-bool dump_ast = false;
-bool print_last_result = false;
-static OwnPtr<Line::Editor> editor;
-static int repl_line_level = 0;
+static bool s_dump_ast = false;
```
Comment (sunverwerth) [inline Userland/js.cpp:65]: I think `g_...` would make more sense here.

Comment (linusg) [reply Userland/js.cpp:65]: And then change it to non-`static`? A search for `static .* g_` has only 6 results across the codebase...

Comment (sunverwerth) [reply Userland/js.cpp:65]: The coding style says static members should be prefixed with s_ but these are global variables and not members. Maybe @awesomekling  should have a look at it.

Comment (awesomekling) [reply Userland/js.cpp:65]: Yeah the `s_` prefix convention is really only for static data members, but I see we have a lot of static locals prefixed with `s_` as well. Accidental habit I guess :) This convention makes sense to me, so let's roll with it.

**Pull request merged**